bitnami/phabricator/templates/deployment.yaml

```diff
@@ -94,44 +163,53 @@ spec:
             - name: SMTP_PROTOCOL
               value: {{ .Values.smtpProtocol | quote }}
             {{- end }}
+            {{- if .Values.extraEnvVars }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          {{- if or .Values.extraEnvVarsCM .Values.extraEnvVarsSecret }}
+          envFrom:
+            {{- if .Values.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.extraEnvVarsCM "context" $) }}
+            {{- end }}
+            {{- if .Values.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.extraEnvVarsSecret "context" $) }}
+            {{- end }}
+          {{- end }}
           ports:
             - name: http
-              containerPort: 80
+              containerPort: {{ .Values.containerPorts.http }}
             - name: https
-              containerPort: 443
+              containerPort: {{ .Values.containerPorts.https }}
             {{- if .Values.phabricatorGitSSH }}
             - name: ssh
-              containerPort: 22
+              containerPort: {{ .Values.containerPorts.ssh }}
             {{- end }}
-          livenessProbe:
-            httpGet:
-              path: /auth/
-              port: http
-              httpHeaders:
-                - name: Host
-                  value: {{ include "phabricator.host" . | quote }}
-            initialDelaySeconds: 180
-            timeoutSeconds: 5
-            failureThreshold: 6
-          readinessProbe:
-            httpGet:
-              path: /auth/
-              port: http
-              httpHeaders:
-                - name: Host
-                  value: {{ include "phabricator.host" . | quote }}
-            initialDelaySeconds: 30
-            timeoutSeconds: 3
-            periodSeconds: 5
+          {{- if .Values.livenessProbe.enabled }}
+          livenessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.livenessProbe "enabled") "context" $) | nindent 12 }}
+          {{- else if .Values.customLivenessProbe }}
+          livenessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.customLivenessProbe "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.readinessProbe.enabled }}
+          readinessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.readinessProbe "enabled") "context" $) | nindent 12 }}
+          {{- else if .Values.customReadinessProbe }}
+          readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.customReadinessProbe "context" $) | nindent 12 }}
```

Comment:
I have added some comments in the values.yaml regarding this, but it was decided we would define the entire readiness/livenessProbe in the template, and if users wants to customize anything apart from path/initialDelay/..., they should use customReadiness/LivenessProbe